package.json

```diff
@@ -41,8 +41,6 @@
     "@turf/boolean-point-in-polygon": "^6.5.0",
     "@turf/helpers": "^6.5.0",
     "@turf/length": "^6.5.0",
-    "@turf/line-slice": "^6.5.0",
```

Comment:
Would be interested in issues with turf implementation, impressive to see this now in `route-snapper-ts`, above my skill level but general :+1: to all this!